Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -878,6 +878,12 @@ void TilesetContentManager::loadTileContent(
     if (pParentTile) {
       if (pParentTile->getState() != TileLoadState::Done) {
         loadTileContent(*pParentTile, tilesetOptions);
+
+        // Finalize the parent if necessary, otherwise it may never reach the
+        // Done state.
+        if (pParentTile->getState() == TileLoadState::ContentLoaded) {
+          finishLoading(*pParentTile, tilesetOptions);
```

Comment:
Look sane.

Verified
* finishLoading can be called from this function (looks to be able to be deferred or called immediately)
* finishLoading is still getting called from the same thread that other callers are using it (main thread)
* Adding this call won't cause any infinite recursion